TestCases/unsteady/pitching_naca64a010_euler/pitching_NACA64A010.cfg

```diff
@@ -11,360 +11,142 @@
 
 % ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
 %
-% Physical governing equations (EULER, NAVIER_STOKES,
-%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
-%                               POISSON_EQUATION)
 SOLVER= EULER
-%
-% Specify turbulent model (NONE, SA, SA_NEG, SST)
 KIND_TURB_MODEL= NONE
-%
-% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT)
 MATH_PROBLEM= DIRECT
-%
-% Restart solution (NO, YES)
 RESTART_SOL= NO
 
 % ------------------------- UNSTEADY SIMULATION -------------------------------%
 %
-% Unsteady simulation (NO, TIME_STEPPING, DUAL_TIME_STEPPING-1ST_ORDER, 
-%                      DUAL_TIME_STEPPING-2ND_ORDER)
 TIME_MARCHING= DUAL_TIME_STEPPING-2ND_ORDER
-%
-% Time Step for dual time stepping simulations (s)
 TIME_STEP= 0.0023555025613149587
 % 24 steps per period: 0.0024536485013697488
-% 25 steps per period: 0.0023555025613149587
 %
-% Total Physical Time for dual time stepping simulations (s)
 MAX_TIME= 0.59
-% 10 periods: 0.5888756403287397
```

Comment:
Yes for these unusual cases it may make sense to keep more comments.

Reply:
👍